### DISCUSSION TITLE
[Alpha] Coachmark

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#2575] [Komponenter]
## Velkommen til vurdering av Coachmark 🙌
`Coachmark` brukes til å forklare brukere hva deler av en løsning gjør, eller hva det betyr. Forklaringene kan settes opp i en serie eller frittstående som bruker aktiverer selv.

Forslaget her er å tilby `Coachmark` som en måte å introdusere nye brukere til en løsning, eller introdusere ny funksjonalitet i en løsning.
<img width="1004" alt="Skjermbilde 2023-12-21 kl  11 03 42" src="https://github.com/navikt/aksel/assets/82579704/235592c8-71a6-48d9-bed1-1fdf3d6f9df6">

Vi har lagt et eksempel på komponenten [Coachmark i Alpha UI kit](https://www.figma.com/file/Jw3DCrpJgK0uiG7v92Wdnk/Alpha-UI-kit?type=design&node-id=1891-453&mode=design) som dere kan teste ut.

For å finne ut om vi skal inkludere `Coachmark` som en komponent i Aksel trenger vi deres hjelp. **Vi er på jakt etter bevis på behov i løsninger som er i prod/demo**. Vi ønsker skjermbilder (uten sensitive opplysninger) og en forklaring av konteksten.

Vi holder ikke aktivt øye med denne diskusjonen, så gi oss et hint om du legger til noe her. Når vi har **fått bevis på minst 3 tilfeller av bruk** vil vi gå videre med komponenten. Del dette gjerne med team du tror `Coachmark` vil passe for.

**Comment by Trent-NAV** [comment]
@sjur-gr Her er en liten prototype fra Pesys. Jeg ser absolutt behovet for Coachmark. Veldig praktisk å kunne angi funksjonsfornyelser og generelle endringer direkte i aktuell kontekst som demonstrert i prototypen, også greit for nye saksbehandlere som kommer til. Hvordan man implementer dette i forhold til frekvens og varighet vil nok variere og justeres etter innsikt og brukserfaring. Jeg ville tro at andre saksbehandlingssystemer også vil kunne benytte dette på lignende måte 👍

https://github.com/navikt/aksel/assets/86958516/374e6586-55da-4208-b148-822f0073af5a
